speech/unit_tests/test_client.py

```diff
@@ -33,15 +26,26 @@ def _make_result(alternatives):
                 ) for alternative in alternatives
             ],
         )
+    else:
+        return cloud_speech_pb2.SpeechRecognitionResult(alternatives=[])
 
-    def _make_sync_response(self, *results):
-        from google.cloud.grpc.speech.v1beta1 import cloud_speech_pb2
 
+def _make_sync_response(*results):
+    from google.cloud.grpc.speech.v1beta1 import cloud_speech_pb2
+    if results:
         response = cloud_speech_pb2.SyncRecognizeResponse(
             results=results,
         )
+    else:
+        response = cloud_speech_pb2.SyncRecognizeResponse()
+    return response
 
-        return response
+
+class TestClient(unittest.TestCase):
+    SAMPLE_RATE = 16000
+    HINTS = ['hi']
```

Comment:
Python tip: don't use mutables (`list` here) as class attributes unless you meant to do it on purpose. A `tuple` would be better here. (It's a test, so no need to change if you don't feel like it, but it's a good habit to get into.)